io-actions/src/fragment/fetch.js

```diff
@@ -1,11 +1,9 @@
 const fetch = require('node-fetch');
-
-async function main({ id, locale }) {
+const { odinId } = require('./paths.js');
```

Comment:
I wonder as a general rule we should be using `mjs` format and use `import` instead of `require`

Reply:
i switched every test back from mjs :) i like that it reminds we are working on node.js code.

Reply:
we could run & debug these tests in WTR with import, but not with require.
Not sure if that makes sense to test backend APIs though.

Reply:
in a browser? :) not really no

Reply:
i'd keep require for now